src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -58,17 +58,17 @@
     {
         "hostname": "www.va.gov",
         "pathnameBeginning": "/",
-        "cookieOnly": true
+        "cookieOnly": false
     },
     {
         "hostname": "www.cem.va.gov",
         "pathnameBeginning": "/",
-        "cookieOnly": true
+        "cookieOnly": false
     },
     {
         "hostname": "benefits.va.gov",
```

Comment:
Why is benefits.va.gov on both line 9 and 69?

Reply:
fixed